docs/_tutorials/curl.md

```diff
@@ -56,7 +59,7 @@ curl -X POST -u user:password solacevmr:8080/SEMP/v2/config/msgVpns/default/clie
   -d '{"clientUsername":"tutorialUser","enabled":true}'
 ```
 
-When you execute this command, if successful the Solace message router will respond with contents that resembles the following:
+When you execute this command, if successful the Solace messaging router will respond with contents that resembles the following:
```

Comment:
Needs further fixing. Should be "if successful the Solace Messaging system will respond with content..." or something similar.